lib/web_ui/lib/src/engine/view_embedder/flutter_view_manager.dart

```diff
@@ -3,10 +3,13 @@
 // found in the LICENSE file.
 import 'dart:async';
 import 'package:ui/src/engine.dart';
+import 'package:ui/ui.dart' show ViewFocusEvent, ViewFocusState;
 
 /// Encapsulates view objects, and their optional metadata indexed by `viewId`.
 class FlutterViewManager {
-  FlutterViewManager(this._dispatcher);
+  FlutterViewManager(this._dispatcher) {
+    _init();
```

Comment:
Is there a reason for extracting this into an `_init()` method?

Reply:
Nothing strictly needed. Just wanted to have it next to `dispose` to reduce the probability of forgetting cleaning up the listeners.